db/seeds/dummy_structures.rb

```diff
@@ -31,10 +31,12 @@
   end
 
   if EarlyCareerTeacherProfile.none?
-    user = User.find_or_create_by!(email: "early-career-teacher@example.com") do |u|
-      u.full_name = "ECT User"
-      u.confirmed_at = Time.zone.now.utc
+    CoreInductionProgramme.all.each_with_index do |cip, index|
+      user = User.find_or_create_by!(email: "early-career-teacher-#{index + 1}@example.com") do |u|
```

Comment:
ah, that's what the indexes are for :/ 
yeah, I think we want course name in the dummy email, even if we repeat this code 4 times

Reply:
Well, you could always add a helper method to commonise it.